core/app/models/spree/variant.rb

```diff
@@ -385,6 +385,23 @@ def display_image(fallback: true)
       images.first || (fallback && product.variant_images.first) || Spree::Image.new
     end
 
+    # Images that can be used for the variant.
+    #
+    # Will first retrieve images on the variant. If it is an empty array,
+    # it'll fallback to master variant images.
+    # @return [Enumerable<Spree::Image>] images for the variant
+    def display_images
+      imgs = images
+      return imgs if !imgs.empty?
+
+      if product && product.master && !is_master?
+        imgs = product.master.images
```

Comment:
I can understand the reasoning to create a new variable in here but I think the name is too generic and can cause confusion with the `images` variable, something like `master_image` might be better. WDYT?

Reply:
Agreed.